pkg/restapi/models/models.go

```diff
@@ -0,0 +1,13 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package models
+
+// KeystoreConfiguration represents a keystore configuration.
+type KeystoreConfiguration struct {
+	Sequence   int    `json:"sequence"`
```

Comment:
what's the use for this sequence? Is it an identifier supplied by the client?

Reply:
Plan to use it when updates are performed to the keystore - client is supposed to set 0 when the keystore is created and the next positive number during updates. If two updates are performed concurrently then `sequence` value can tell who wins - kind of optimistic concurrency control.

Reply:
call it `Version` then

Reply:
@aholovko you cannot count on the client maintaining state (think of the browser).

Reply:
> call it `Version` then

agree that perhaps version would be more descriptive but it looks like that `sequence` is something that is adopted here - it is used in [EDV spec](https://digitalbazaar.github.io/encrypted-data-vaults/#datavaultconfiguration) and also digitalbazaar uses it as well in their kms (e.g. [here](https://github.com/digitalbazaar/bedrock-kms/blob/3ae50ddf777487bcdf086bcc06312d70339d5863/lib/keystores.js#L172))

Reply:
> @aholovko you cannot count on the client maintaining state (think of the browser).

it might work in the following way: before updating keystore configuration (let say adding additional controller or updating existing one (that might be considered as transferring keystore to other owner)) you know what is the current keystore configuration (e.g. by querying it) - you make your changes and increment `sequence` by one - no need to store state here

Reply:
@aholovko ok. Sounds like what CouchDB does.